### ISSUE TITLE
[Bug]: Cant Copy Game ID and cant Start Spoofing

### ISSUE THREAD

**Issue by GGCBlade** [ItsLogic/Xbox-Achievement-Unlocker#220]
### What version of the Xbox Achievement Unlocker were you using (you can see this in the settings page)?

2.5

### Tool Section

Games

### What is the bug?

If i want to cpoy the ID via Games it comes with error and if i want to try to Start Spoofing nothing happens

Information
=================================
Tool Version: 2.5.0.0
System Version: 10.0.22631.0
Tool Uptime: 00:01:18.5639492
=================================
Exception
=================================
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at XAU.ViewModels.Pages.GamesViewModel.CopyToClipboard(String index)
   at XAU.Views.Pages.GamesPage.ButtonBase_RightClick(Object sender, MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by TheTitanBr** [comment]
Same thing here

**Comment by ItsLogic** [comment]
fixed in 2.5.1
